Odysee/Controllers/Content/HomeViewController.swift

```diff
@@ -94,13 +100,10 @@ class HomeViewController: UIViewController, UITableViewDelegate, UITableViewData
         assert(Thread.isMainThread)
         result.showErrorIfPresent()
         if case let .success(payload) = result {
-            UIView.performWithoutAnimation {
-                claimListView.performBatchUpdates {
-                    let oldCount = claims.count
-                    claims.append(contentsOf: payload.items)
-                    let indexPaths = (oldCount..<claims.count).map { IndexPath(item: $0, section: 0) }
-                    claimListView.insertRows(at: indexPaths, with: .automatic)
-                }
+            let oldCount = claims.count
+            claims.append(contentsOf: payload.items)
+            if claims.count != oldCount {
+                claimListView.reloadData()
```

Comment:
I went back to using `reloadData` here because it maintains the scrolling momentum better than `insertRows`.